src/microbe_stage/FloatingChunk.cs

```diff
@@ -75,6 +75,16 @@ public class FloatingChunk : RigidBody, ISpawned, ISaveLoadedTracked
     /// </summary>
     public float Damages { get; set; }
 
+    /// <summary>
+    ///   When true, the chunk will despawn when the despawn timer finishes
+    /// </summary>
+    public bool UsesDespawnTimer { get; private set; }
```

Comment:
by the looks of it, this prop seems pretty useless, because this is always the inverse of `Dissolves`.
Either remove this property and use `!Dissolves` on line 230, or change this to something like this:
```suggestion
    public bool UsesDespawnTimer => !Dissolves;
```

Reply:
I commented related to this on discord / on here before?
I think this property should be kept, but it should be set from the spawn helpers methods, instead of by a not very clear condition inside the FloatingChunk class.

Reply:
I've just read your comments above, haven't noticed them before. When the setter becomes public, it should be kept.